app/deepseg.cc

```diff
@@ -613,11 +626,24 @@ int main(int argc, char* argv[]) try {
 		loopback_free(lbfd);
 	});
 
-
 	// Processing components, all at capture true geometry
-	cv::Mat mask(capGeo.value().second, capGeo.value().first, CV_8U);
+	std::optional<std::pair<size_t, size_t>> mask_dim = capGeo;
+	if ( crop_region.height ) {
+		mask_dim = {crop_region.width, crop_region.height};
+	}
+	cv::Mat mask(mask_dim.value().second, mask_dim.value().first, CV_8U);
```

Comment:
```suggestion
	std::pair<size_t, size_t> mask_dim = *capGeo;
	if (crop_region.height) {
		mask_dim = {crop_region.width, crop_region.height};
	}
	cv::Mat mask(mask_dim.second, mask_dim.first, CV_8U);
```

Same remark as above re `cv::Size`.